go.mod

```diff
@@ -228,3 +228,5 @@ require (
 )
 
 replace github.com/opencontainers/runc => github.com/opencontainers/runc v1.1.1-0.20240131200429-02120488a4c0
+
+replace github.com/opencontainers/runtime-spec => github.com/opencontainers/runtime-spec v1.1.1-0.20230823135140-4fec88fd00a4
```

Comment:
I don;'t think this is needed any longer.

Reply:
Hopefully; let’s see the current commit’s tests outcomes, first.

Reply:
Removed.